app/namespace.go

```diff
@@ -298,9 +299,14 @@ func (c *NamespaceClient) addRegion(ctx *cli.Context) error {
 		return fmt.Errorf("namespace cloud provider is required")
 	}
 
+	targetRegion := fmt.Sprintf("%s-%s", cloudProvider, region)
```

Comment:
Do we already have a cloud provider flag for tcld?

Reply:
yes. I added a optional one.